### ISSUE TITLE
PlaceAutocompleteRequest and PlaceDetailsRequest are missing required param key

### ISSUE THREAD

**Issue by kf6kjg** [googlemaps/google-maps-services-js#339]
`"@googlemaps/google-maps-services-js": "^2.0.2"`

* https://developers.google.com/places/web-service/autocomplete
* https://developers.google.com/places/web-service/details

Both specify the parameter `key` as a required field.  This field is missing from the PlaceAutocompleteRequest and PlaceDetailsRequest interfaces, and consequently from the subsequent process.  Looks like the refactor that got most of the rest of this library using the new key parameter missed at least these two interfaces.

**Comment by henrymingwu** [comment]
@kf6kjg it should be included on the interface, see https://github.com/googlemaps/google-maps-services-js/blob/master/src/places/details.ts#L39

Is it not working for you if you pass `key` as a field in `params`?

**Comment by kf6kjg** [comment]
Looking closer I just realized that the errors are about `sessiontoken` being required on PlaceAutocompleteRequest, see #221 and #323.  From doing a quick code review, wherein I totally missed the `& RequestParams` operation, and assuming that my editor was complaining about `key`, I found that details.ts looked the same and thus assumed it would have the same problem I thought I was observing.  Such is the result of assuming.

Now that I see my mistake, I'm closing this and assuming that the code will work once #323 is fixed. :P
